usr/lib/cca_stdll/cca_specific.c

```diff
@@ -3327,6 +3355,135 @@ CK_RV token_specific_set_attrs_for_new_object(STDLL_TokData_t *tokdata,
 }
 #endif /* NO_PKEY */
 
+typedef struct {
+    CK_BBOOL card_level_set;
+    struct cca_version min_card_version;
+} cca_min_card_version_t;
+
+/* return -1 if v1 < v2, 0 if v1 == v2, 1 if v1 > v2 */
+static int compare_cca_version(const struct cca_version *v1,
+                               const struct cca_version *v2)
+{
+
```

Comment:
remove empty line (yes I know it was there also in EP11's version of this function).